iaso/tests/api/test_instances.py

```diff
@@ -983,7 +983,7 @@ def test_stats(self):
                     "Hydroponic public survey": 1,
                     "Hydroponic public survey III": 1,
                     "Hydroponics study": 4,
-                    "index": "2020-02-01T00:00:00.000Z",
+                    "index": "2020-02-01T00:00:00.000",
```

Comment:
This is related to the `/api/instances/stats/` endpoint and somewhat linked to [this change in Pandas](https://pandas.pydata.org/docs/whatsnew/v1.5.0.html#serializing-tz-naive-timestamps-with-to-json-with-iso-dates-true). I didn't find the exact reason though…